packages/assets-controllers/src/AccountTrackerController.ts

```diff
@@ -1,4 +1,6 @@
+import assert from 'assert';
```

Comment:
`@metamask/utils` has an assert function which does not rely on Node.js builtins.

Reply:
Thanks! Updated in c11c27ed